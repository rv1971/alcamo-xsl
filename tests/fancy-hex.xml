<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <xsl:stylesheet
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xml:id="xslt"
      version="1.0">
    <xsl:import href="../xsl/fancy.xsl"/>

    <xsl:template match="/">
      <a:result>
        <xsl:apply-templates select="/a:doc/a:data"/>
      </a:result>
    </xsl:template>

    <xsl:template match="a:item">
      <a:item>
        <xsl:value-of select="."/>
        <xsl:value-of select="' '"/>
        <xsl:apply-templates select="." mode="a:hex"/>
      </a:item>
    </xsl:template>
  </xsl:stylesheet>

  <a:data>
    <a:item>0</a:item>
    <a:item>1</a:item>
    <a:item>9</a:item>
    <a:item>10</a:item>
    <a:item>11</a:item>
    <a:item>12</a:item>
    <a:item>13</a:item>
    <a:item>14</a:item>
    <a:item>15</a:item>
    <a:item>16</a:item>
    <a:item>17</a:item>
    <a:item>18</a:item>
    <a:item>31</a:item>
    <a:item>32</a:item>
    <a:item>33</a:item>
    <a:item>255</a:item>
    <a:item>256</a:item>
    <a:item>257</a:item>
    <a:item>65535</a:item>
    <a:item>65536</a:item>
    <a:item>4294967295</a:item>
    <a:item>4294967296</a:item>
    <a:item>4294967297</a:item>
  </a:data>
</a:doc>
