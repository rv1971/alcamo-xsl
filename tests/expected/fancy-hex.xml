<?xml version="1.0"?>
<a:result xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <a:item>0 0</a:item>
  <a:item>1 1</a:item>
  <a:item>9 9</a:item>
  <a:item>10 A</a:item>
  <a:item>11 B</a:item>
  <a:item>12 C</a:item>
  <a:item>13 D</a:item>
  <a:item>14 E</a:item>
  <a:item>15 F</a:item>
  <a:item>16 10</a:item>
  <a:item>17 11</a:item>
  <a:item>18 12</a:item>
  <a:item>31 1F</a:item>
  <a:item>32 20</a:item>
  <a:item>33 21</a:item>
  <a:item>255 FF</a:item>
  <a:item>256 100</a:item>
  <a:item>257 101</a:item>
  <a:item>65535 FFFF</a:item>
  <a:item>65536 10000</a:item>
  <a:item>4294967295 FFFFFFFF</a:item>
  <a:item>4294967296 100000000</a:item>
  <a:item>4294967297 100000001</a:item>
</a:result>
